src/main/resources/alfresco/messaging-context.xml

```diff
@@ -39,4 +39,13 @@
             </list>
         </property>
     </bean>
+
+    <bean id="messagingObjectMapper" class="org.gytheio.messaging.jackson.ObjectMapperFactory"
```

Comment:
Can we swap that to [Jackson2ObjectMapperFactoryBean](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/http/converter/json/Jackson2ObjectMapperFactoryBean.html) from Spring?

Are we ok with reusing the Messaging subsystem from Desktop Sync?
I presume Camel configs are shared as well.

Reply:
I added Jackson2ObjectMapperFactoryBean, thanks for the recommendation, we removed the use of gytio now.